library/Zend/Validate/File/MimeType.php

```diff
@@ -161,6 +161,7 @@ public function __construct($mimetype)
      * magic file.
      *
      * @return string
+     * @throws Zend_Validate_Exception
```

Comment:
this throws is bogus, it's never thrown (but caught and ignored silently)

will be removed when porting https://github.com/zf1s/zf1/pull/54

Reply:
https://github.com/Shardj/zf1-future/pull/265